sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -107,8 +123,37 @@ def _create_appconfig_pipeline(self, **kwargs):
                 self._config.headers_policy,
                 self._config.user_agent_policy,
                 AppConfigRequestsCredentialsPolicy(self._config.credentials),
+                self._config.retry_policy,
                 SyncTokenPolicy(),
+                self._config.logging_policy,  # HTTP request/response log
+                DistributedTracingPolicy(**kwargs),
+                HttpLoggingPolicy(**kwargs)
+            ]
+
+        if not transport:
+            transport = RequestsTransport(**kwargs)
+
+        return Pipeline(transport, policies)
+
+    def _create_appconfig_aad_pipeline(self, base_url, credential, **kwargs):
+        transport = kwargs.get('transport')
+        policies = kwargs.get('policies')
+        if base_url.endswith('/'):
+            scope = base_url + '.default'
+        else:
+            scope = base_url + '/.default'
```

Comment:
maybe personal pref but I try to avoid conditionals for things like this:
```
scope = base_url.strip("/") + "/.default"
```
Also nit: I think Black will want double quotes on the literal